cmd/signozschemamigrator/schema_migrator/squashed_logs_migrations.go

```diff
@@ -0,0 +1,685 @@
+					Columns: []Column{
+						{Name: "timestamp", Type: DateTimeColumnType{}, Codec: "ZSTD(1)"},
+						{Name: "tagKey", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+						{Name: "tagType", Type: EnumerationColumnType{Values: []string{"'tag' = 1", "'resource' = 2", "'scope' = 3"}, Size: 8}, Codec: "ZSTD(1)"},
+						{Name: "tagDataType", Type: EnumerationColumnType{Values: []string{"'string' = 1", "'bool' = 2", "'int64' = 3", "'float64' = 4"}, Size: 8}, Codec: "ZSTD(1)"},
+						{Name: "stringTagValue", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "int64TagValue", Type: NullableColumnType{ColumnTypeInt64}, Codec: "ZSTD(1)"},
+						{Name: "float64TagValue", Type: NullableColumnType{ColumnTypeFloat64}, Codec: "ZSTD(1)"},
+					},
+					Engine: Distributed{
+						Database:    "signoz_logs",
+						Table:       "tag_attributes",
+						ShardingKey: "rand()",
+					},
+				},
+			},
+			DownItems: []Operation{
+				DropTableOperation{
+					Database: "signoz_logs",
+					Table:    "distributed_tag_attributes",
+				},
+			},
+		},
+		{
+			MigrationID: 14,
+			UpItems: []Operation{
+				CreateTableOperation{
+					Database: "signoz_logs",
+					Table:    "usage",
+					Columns: []Column{
+						{Name: "tenant", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "collector_id", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "exporter_id", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "timestamp", Type: DateTimeColumnType{}, Codec: "ZSTD(1)"},
+						{Name: "data", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					},
+					Engine: MergeTree{
+						OrderBy: "(tenant, collector_id, exporter_id, timestamp)",
+						TTL:     "timestamp + toIntervalDay(3)",
+						Settings: TableSettings{
+							{Name: "index_granularity", Value: "8192"},
+						},
+					},
+				},
+			},
+			DownItems: []Operation{
+				DropTableOperation{
+					Database: "signoz_logs",
+					Table:    "usage",
+				},
+			},
+		},
+		{
+			MigrationID: 15,
+			UpItems: []Operation{
+				CreateTableOperation{
+					Database: "signoz_logs",
+					Table:    "distributed_usage",
+					Columns: []Column{
+						{Name: "tenant", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "collector_id", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "exporter_id", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+						{Name: "timestamp", Type: DateTimeColumnType{}, Codec: "ZSTD(1)"},
+						{Name: "data", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					},
+					Engine: Distributed{
+						Database:    "signoz_logs",
+						Table:       "usage",
+						ShardingKey: "cityHash64(rand())",
+					},
+				},
+			},
+			DownItems: []Operation{
+				DropTableOperation{
+					Database: "signoz_logs",
+					Table:    "distributed_usage",
+				},
+			},
+		},
+		{
+			MigrationID: 16,
+			UpItems: []Operation{
+				CreateTableOperation{
+					Database: "signoz_logs",
+					Table:    "logs_v2",
```

Comment:
it's replicated by default for logs_v2 and logs_v2_resource

Reply:
For every new user, replication will be enabled by default and controlled by that. It doesn't make any difference to hard code it here.